extension/httpforwarder/testdata/config.yaml

```diff
@@ -0,0 +1,8 @@
+http_forwarder:
+  ingress:
+    endpoint: localhost:7070
+  egress:
+    endpoint: http://target/
+    headers:
+      otel_http_forwarder: dev
+    timeout: 5s
```

Comment:
Can you please submit another PR to add this file along with a test case similar to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11075